pkg/version/version.go

```diff
@@ -41,14 +43,24 @@ func Get() Info {
 		gitCommit = "dev"
 	}
 
-	return Info{
+	result := Info{
 		GitVersion: gitVersion,
 		GitCommit:  gitCommit,
 		BuildDate:  buildDate,
 		GoVersion:  runtime.Version(),
 		Compiler:   runtime.Compiler,
 		Platform:   fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
+
+	if info, ok := debug.ReadBuildInfo(); ok {
```

Comment:
I haven't looked into it.. but I'm also questioning if this complexity of reading build info at runtime is necessary.  It seems clever... but it is a very indirect way to have a dependency.  not that this is likely to happen... but what if the dependency is removed or the naming changes...   This is feel if this is the only way.. but is there a way to get the version directly from client-go?

Reply:
I don't think there is a way to get the version directly from `client-go`. There is a `version` package inside, but using
`version.Get().GitVersion` results in a string of `"v0.0.0-master+$Format:%h$"`. 

From what I read is the `debug.ReadBuildInfo()` the correct way to get information about the used modules.